aries_cloudagent/vc/vc_di/cryptosuites/__init__.py

```diff
@@ -0,0 +1,11 @@
+from .eddsa_jcs_2022 import EddsaJcs2022
+
+CONTEXTS = {"data-integrity-v2": "https://w3id.org/security/data-integrity/v2"}
```

Comment:
When would we look up this context by the string `data-integrity-v2`? I get the feeling it might be simpler to just have `DIV2_CONTEXT = "..."`.

Reply:
I agree, not sure why I settled for this.

I would like to create a file similar to [this file](https://github.com/hyperledger/aries-cloudagent-python/blob/main/aries_cloudagent/vc/ld_proofs/constants.py), but focused on contexts with their digests. 

I'm just not convinced by the current structure of the vc directory. I think we should think about the purpose of this directory and re-organize how files are structured. I would like to create a linked-data sub directory where the document loaders and context uri/files live, and manage all linked-data related operations in there, providing some functions to detect dropped terms, etc...

Then just focus on data integrity proofs...